rs/execution_environment/src/execution/upgrade.rs

```diff
@@ -106,13 +106,15 @@ pub(crate) fn execute_upgrade(
 
     // Stage 0: validate input.
     if let Err(err) = helper.validate_input(&original, &round, round_limits) {
+        let is_wasm64_execution = original.is_wasm64_execution;
```

Comment:
Nit: We can directly use `original.is_wasm64_execution` below and it's quite readable imo. Now there's an extra line that doesn't bring much.

Reply:
But then can we not simply use `original.is_wasm64_execution` inside `finish_err`?

Reply:
yes, absolutely correct! fixed.